src/components/seconardynavnew/mobile.tsx

```diff
@@ -331,17 +342,19 @@ const MobileView = () => {
                         MongoDB Developer
                     </TypographyScale>
                     {!mobileMenuIsOpen && (
-                        <SystemIcon
-                            sx={chevronStylesForMainLink}
-                            className="chevron-icon"
-                            name={ESystemIconNames.CHEVRON_DOWN}
-                            size="small"
-                            strokeWeight="large"
-                        />
+                        <>
```

Comment:
you might be able to skip the fragment if there's just one child element

Reply:
Nice catch!